addons/popochiu/editor/gizmos/gizmo_clickable_plugin.gd

```diff
@@ -75,6 +85,43 @@ func _forward_canvas_gui_input(event: InputEvent) -> bool:
 #region Signals handlers
 func _on_property_changed(property: String):
 	update_overlays()
+
+
+func _on_gizmo_settings_changed() -> void:
+	var gizmo_id = 0
+	var default_font = EditorInterface.get_editor_theme().default_font
+
+	for gizmo in _gizmos:
+		match gizmo_id:
+			WALK_TO_POINT:
+				gizmo.set_theme(
```

Comment:
You are introducing a bit of redundant code at this level, invoking `set_theme()` explicitly.

Since `_init_gizmo()` function sets all the plugin params and info in a single pass, can't you just re-invoke it here?

Of course, some properties will be re-assigned, but they should not change and the code would be more consistent.

Reply:
Nevermind, this would create a new gizmo altogether.